eagleEvents/templates/add-update-event.html.j2

```diff
@@ -3,41 +3,47 @@
+                                      <option value="{{ o.id }}" selected>{{ o.name }}</option>
+                                  {% else %}
+                                      <option value="{{ o.id }}">{{ o.name }}</option>
+                                  {% endif %}
+                              {% endfor %}
                             </select>
                     </div>
                     <div class="form-group row">
                         <label for="exampleSelect1">Event Planner</label>
-                            <select class="form-control form-control-event" id="eventPlanner">
-                              <option>Event Planner 1</option>
-                              <option>Event Planner 2</option>
-                              <option>Event Planner 3</option>
-                              <option>Event Planner 4</option>
-                              <option>Event Planner 5</option>
+                            <select class="form-control form-control-event" name="planner">
+                                {% for o in planners %}
+                                    {%  if o.id == planner.id %}
+                                        <option value="{{ o.id }}" selected>{{ o.name }}</option>
+                                    {% else %}
+                                        <option value="{{ o.id }}">{{ o.name }}</option>
+                                    {% endif %}
+                                {% endfor %}
                             </select>
                     </div>
-                </form>
+                </div>
                        <div class="buttons-event">
-                           <button method=post enctype=multipart/form-data class="btn btn-lg btn-success btn-block btn-event-function"
-                                       onclick="document.getElementById('file').click();" type="submit">Import Guests</button>
-                           <button class="btn btn-lg btn-success btn-block btn-event-function" type="submit">Seating Chart</button>
-                           <button class="btn btn-lg btn-success btn-block btn-event-function" type="submit">Print Attendance List</button>
-                           <button class="btn btn-lg btn-success btn-block btn-event-function" type="submit">Print Table Cart</button>
-                           <form method="post" enctype="multipart/form-data">
-                               <input type=file id="file" name="file" onchange="this.form.submit()" style="display:none;"/>
-                           </form>
+                           <button class="btn btn-lg btn-success btn-block btn-event-function"
+                                       onclick="document.getElementById('file').click();" type="button" name="button" value="import">Import Guests</button>
+                           <button class="btn btn-lg btn-success btn-block btn-event-function" type="submit" name="button" value="seat"
```

Comment:
It looks like this button is going to the print seating chart route. I think it's supposed to be going to the seating chart page. The seating chart page will have a print button there.

Reply:
Good catch. I just saw a method dealing with the seating chart and went with it. I had the url_for done incorrectly there too...